src/components/Pressroom/Timeline/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { Box, Button, Step, StepIcon, StepLabel, Stepper, type Theme, Typography, useMediaQuery } from '@mui/material'
+import MobileStepper from '@mui/material/MobileStepper'
+import { useState } from 'react'
+import LessThanIcon from '@/public/images/less-than.svg'
+import GreatThanIcon from '@/public/images/great-than.svg'
+import FullCircleIcon from '@/public/images/circle-full.svg'
+import EmptyCircleIcon from '@/public/images/circle-empty.svg'
+import { type Entry } from 'contentful'
+import { type TypeSimpleBaseBlockSkeleton } from '@/contentful/types'
+import css from './styles.module.css'
+
+const STEPS_PER_DOT_SM = 1
+const STEPS_PER_DOT_LG = 3
+
+const StepCard = ({ title, text }: { title: string; text: string }) => (
+  <div className={css.card}>
+    <Typography variant="h4" mb="16px">
+      {title}
+    </Typography>
+    <Typography>{text}</Typography>
+  </div>
+)
+
+type TimelineProps = { items: Entry<TypeSimpleBaseBlockSkeleton, undefined, string>[] }
+
+const Timeline = ({ items }: TimelineProps) => {
+  const [activeStep, setActiveStep] = useState(0)
+  const isSmallScreen = useMediaQuery((theme: Theme) => theme.breakpoints.down('sm'))
+
+  const STEPS_PER_DOT = isSmallScreen ? STEPS_PER_DOT_SM : STEPS_PER_DOT_LG
+  const stepsNumber = Math.ceil(items.length / STEPS_PER_DOT)
```

Comment:
Nit: this isn't a constant.

```suggestion
  const stepsPerDot = isSmallScreen ? STEPS_PER_DOT_SM : STEPS_PER_DOT_LG
  const stepsNumber = Math.ceil(items.length / STEPS_PER_DOT)
```